vowpal_wabbit/manylinux-2010/manylinux2010-build.Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM quay.io/pypa/manylinux2010_x86_64:latest
```

Comment:
who mantains this image?

Reply:
like how trustworthy is it

Reply:
This is one of the standard manylinux images - I believe it is trustworthy. https://github.com/pypa/manylinux